robottelo/cli/factory.py

```diff
@@ -2679,14 +2681,30 @@ def setup_capsule_virtual_machine(capsule_vm, org_id=None, lce_id=None,
         except CLIReturnCodeError as err:
             raise CLIFactoryError(
                 u'Failed to fetch repository info\n{0}'.format(err.msg))
-    # Synchronize the RH repositories
-    for rh_repo in rh_repos:
+    # If we aren't working with CDN, create custom repo with latest capsule
+    # repo available
+    if not settings.cdn and settings.capsule_repo:
```

Comment:
think we should raise exception when not settings.cdn and not settings.capsule_repo